README.md

```diff
@@ -125,7 +153,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema describes how a database is constructed. In a database, a entries on one table would match multiple tables on another database through the primary key (foreign key on the other table).
 ```
 
```

Comment:
the foreign keys would match the ids of *entries* (a row in a table) in other tables. your def of schema is 💯 correct tho; more specifically it defines what columns in a table